checkstyle.xml

```diff
@@ -4,6 +4,10 @@
         "http://www.puppycrawl.com/dtds/configuration_1_2.dtd">
 <module name="Checker">
 
+    <module name="SuppressionFilter">
+        <property name="file" value="${config_loc}/checkstyle-suppressions.xml"/>
+    </module>
```

Comment:
This is hell, eclipse checkstyle plugin knows about `config_loc` and it is variable that tells in absolute path, it is said that it is added in IntelliJ and it should work. Here is nice [blog](https://jgonian.wordpress.com/2010/12/12/maven-checkstyle-and-eclipse/) wrap-up.

However maven checkstyle plugin does not know about this variable and that is reason why we need to define it via `propertyExpansion` tag.

@nivancevic @mgobec we have two options:
1. Leave it like this and it works in Travis, Command line and Eclipse (still to be tested in IntelliJ) and have pom be dependent on checkstyle eclipse plugin (not that great)
2. Have only relative path which is good for maven plugin and it knows how to resolve it but we are than without IDE support (it is more clear and folder and IDE independent but IMO having IDE go yellow and red is much useful and keeps project clean)

What do you think?